app/(candidate)/dashboard/polls/create/CreatePoll.tsx

```diff
@@ -491,10 +500,18 @@ const AudienceSelectionForm: React.FC<{
       <H1 className="md:text-center">
         How many constituents do you want to message?
       </H1>
-      <p className="text-left md:text-center mt-4 mb-8 text-lg font-normal text-muted-foreground">
-        There are {numberFormatter(query.data.totalConstituents)} constituents
-        with cell phone numbers in your community.
-      </p>
+      <div className="flex flex-col mt-4 mb-8">
+        <p className="text-left md:text-center text-lg font-normal text-muted-foreground">
+          There are {numberFormatter(rawTotalConstituents)} constituents with
+          cell phone numbers in your community.
+        </p>
+        {isTotalConstituentsCapped && (
+          <p className="text-left md:text-center text-lg font-normal text-muted-foreground">
+            You can only send to {numberFormatter(MAX_CONSTITUENTS_PER_RUN)}{' '}
+            constituents at a time.
+          </p>
+        )}
```

Comment:
The message that should be conditionally rendered is actually already on this page, on new line 551